src/script/deploy.js

```diff
@@ -295,22 +383,20 @@ async function main() {
 
   program
     .name("deploy")
-    .description("A CLI interface for deploying the WorldID state bridge on Linea.")
-    .option("--no-config", "Do not use any existing configuration.")
-    .option("--env <environment>", "Specify the environment (e.g., mainnet, sepolia)", "mainnet");
```

Comment:
This supposed to be a feature (mentioned before) :)
Do you think it was a bad idea?